src/clr-angular/forms/common/label.ts

```diff
@@ -27,6 +42,12 @@ export class ClrLabel implements OnInit, OnDestroy {
   private subscriptions: Subscription[] = [];
   private enableGrid = true;
 
+  get labelText(): string {
+    if (isPlatformBrowser(this.platformId)) {
+      return this.el.nativeElement.textContent;
```

Comment:
We don't really need to check platform here. Even if `.textContent` itself turns out to be `undefined`, there is no harm as we don't try to access and use deeper nested properties from `.textContent`.

Reply:
ok. Will remove.